modules/ossm-installing-cert-manager.adoc

```diff
@@ -70,36 +89,102 @@ metadata:
 spec:
   ca:
     secretName: istio-ca
----
 ----
-+
+
 .. Create the objects by running the following command:
 +
 [source, terminal]
 +
 ----
 $ oc apply -f issuer.yaml
 ----
-+
+
 .. Wait for the `istio-ca` certificate to contain the "Ready" status condition by running the following command:
 +
 [source, terminal]
 ----
 $ oc wait --for=condition=Ready certificates/istio-ca -n istio-system
 ----
 
-. Copy the `istio-ca` certificate to the `cert-manager` namespace so it can be used by istio-csr:
+. Create the `IstioCSR` custom resource:
+
+.. Create the `IstioCSR` custom resource similar to the following example:
++
+.Example `istioCSR.yaml` file
+[source, yaml]
+----
+apiVersion: operator.openshift.io/v1alpha1
+kind: IstioCSR
+metadata:
+  name: default
+  namespace: istio-csr
+spec:
+  istioCSRConfig:
+    certManager:
+      issuerRef:
+        name: istio-ca
+        kind: Issuer
+        group: cert-manager.io
+    istiodTLSConfig:
+      trustDomain: cluster.local
+    istio:
+      namespace: istio-system
+----
+
+.. Create the `istio-csr` agent by by running the following command:
 +
-.. Copy the secret to a local file by running the following command:
+[source, terminal]
++
+----
+$ oc create -f istioCSR.yaml
+----
+
+.. Verify if the `istio-csr` deployment is ready by running the following command:
 +
 [source, terminal]
++
+----
+$ oc get deployment -n istio-csr
+----
+
+. Install the `istio` resource:
++
+[NOTE]
+====
+The configuration disables the built-in CA server for {istio} and forwards certificate signing requests from `istiod` to `istio-csr`. The `istio-csr` agent obtains certificates for both `istiod` and mesh workloads from cert-manager. The istiod TLS certificate generated by `istio-csr` is mounted into the pod at a known location for use.
+====
+
+.. Create the `{istio}` object similar to the following example:
++
+.Example `istio.yaml` file
+[source, yaml]
 ----
-$ oc get -n istio-system secret istio-ca -o jsonpath='{.data.tls\.crt}' | base64 -d > ca.pem
+apiVersion: sailoperator.io/v1
+kind: Istio
+metadata:
+  name: default
+spec:
+  version: v1.24-latest
+  namespace: istio-system
+  values:
+    global:
+      caAddress: cert-manager-istio-csr.istio-csr.svc:443
+    pilot:
+      env:
+        ENABLE_CA_SERVER: "false"
 ----
+
+.. Create the `{istio}` resource by running the following command:
 +
-.. Create a secret from the local certificate file in the `cert-manager` namespace by running the following command:
+[source, terminal]
++
+----
+$ oc apply -f istio.yaml
+----
+
+.. Wait for the `istio` resource to contain the "Ready" status condition by running the following command:
```

Comment:
```suggestion
.. Verify that the `istio` resource displays the "Ready" status condition by running the following command:
```
Is the user entering the command so that they can wait, or are they entering it so that they can verify that the resource is ready?

Reply:
This has been confusing for me to draft too. But I will take your suggestion. :) I know the command is about waiting, but not literally as the user will have an action to enter the command. :D